packages/ragbits-core/src/ragbits/core/vector_stores/qdrant.py

```diff
@@ -146,6 +147,21 @@ async def retrieve(self, vector: list[float], options: VectorStoreOptions | None
             for id, document, vector, metadata in zip(ids, documents, vectors, metadatas, strict=True)
         ]
 
+    @traceable
+    async def remove(self, ids: list[str]) -> None:
+        """
+        Remove entries from the vector store.
+
+        Args:
+            ids: The list of entries' IDs to remove.
+        """
+        await self._client.delete(
```

Comment:
do you know what happens if the collection doesn't exist here? Will qdrant throw an exception or it's just noop?

Reply:
Qdrant will throw a ValueError saying "`Collection {collection_name} not found`"

Reply:
ok, probably we should introduce some unified error handling for vector databases. For now let's just mention it in the docstring that this method raises `ValueError`

Reply:
Docstring updated!